### PR TITLE
(PDB-1939) Preserve command id in enqueue-command

### PR DESCRIPTION
**Pull request by mullr**

### CODE REVIEW

File: test/puppetlabs/puppetdb/command_test.clj [hunk 1]
```diff
@@ -1366,6 +1368,27 @@
                  (get "deactivated")
                  (pt/from-string)))))))
 
+(deftest command-response-channel
+  (svc-utils/with-puppetdb-instance
+    (let [pdb (get-service svc-utils/*server* :PuppetDBServer)
+          dispatcher (get-service svc-utils/*server* :PuppetDBCommandDispatcher)
+          shared-globals (partial cli-svc/shared-globals pdb)
+          enqueue-command (partial enqueue-command dispatcher)
+          globals (shared-globals)
```
Comment (wkalt) [inline test/puppetlabs/puppetdb/command_test.clj]: this can be combined with the shared-globals line above

**Comment by wkalt** [comment]
looks good to me aside from minor note

File: test/puppetlabs/puppetdb/command_test.clj [hunk 2]
```diff
@@ -1366,6 +1368,27 @@
...
+          {{:keys [connection endpoint]} :command-mq} globals
```
Comment (ajroetker) [inline test/puppetlabs/puppetdb/command_test.clj]: This can also be combined with the line above

File: test/puppetlabs/puppetdb/command_test.clj [hunk 3]
```diff
@@ -1366,6 +1368,27 @@
...
+          command-uuid (ks/uuid)]
+      (async/tap response-mult response-chan)
+      (enqueue-command connection endpoint
+                       (command-names :deactivate-node) 3
+                       {:certname "foo.local" :producer_timestamp (java.util.Date.)}
+                       command-uuid)
+      (is (= command-uuid
+             (async/alt!!
+               response-chan ([msg] (:id msg))
+               (async/timeout 2000) ::timeout))))))
```
Comment (ajroetker) [inline test/puppetlabs/puppetdb/command_test.clj]: I might pull this out into a let, at first glance I thought you were comparing a channel to uuid

**Comment by mullr** [comment]
Just corrected the issue numbert.

**Pull request merged**